TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/TypeConverter/NodeConverter.php

```diff
@@ -224,7 +224,7 @@ protected function setNodeProperties($nodeLike, NodeType $nodeType, array $prope
                     }
                 break;
                 case 'DateTime':
-                    if ($nodePropertyValue !== '') {
+                    if ($nodePropertyValue !== '' && \DateTime::createFromFormat(\DateTime::W3C, $nodePropertyValue) !== false) {
```

Comment:
Hm, this call `createFromFormat()` twice. What about assigning the result to a variable and check that instead?